src/styles/stars.css

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-position: center;
+  background-repeat: no-repeat;
+
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+
+  background-image: url("../images/star.svg");
+}
+
+.stars--1 :nth-child(-n + 1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background-image: url("../images/star-active.svg");
+}
```

Comment:
The last right star has a margin, do we need it?
![image](https://user-images.githubusercontent.com/66055069/234004023-6895c17a-3cff-4b04-bbb8-ab92f3455ff9.png)